test_model_configs/single_population_node_with_updaters_config.txt

```diff
@@ -0,0 +1,732 @@
+index: 0
+name: "root"
+branch_node {
+  branches {
+    condition {
+      name: "cookie vs. cookieless. name field is optional."
+      op: EQUAL
+      value: "cookied"
+    }
+    node {
+        name: "cookied"
+        index: 2
+        branch_node {
+          branches {
+            condition {
+              name: "pool"
+              op: EQUAL
+              value: "cookied_pool"
+            }
+            node {
+              name: "cookied_pool"
+              index: 0
+                branch_node {
+                  branches {
+                    condition {
+                      name: "country"
+                      op: EQUAL
+                      value: "GB"
+                    }
+                    node {
+                      name: "cookied_pool-GB"
+                      index: 0
+                      branch_node {
+                        branches {
+                          condition {
+                            name: "acting_user_space"
+                            op: EQUAL
+                            value: "phone"
+                          }
+                          node {
+                            name: "cookied_pool-GB-phone"
+                            index: 0
+                            branch_node {
+                              branches {
+                                condition {
+                                  op: AND
+                                   sub_filters {
+                                    name: "min_age"
+                                    op: GT
+                                    value: "18"
+                                  }
+                                   sub_filters {
+                                    name: "max_age"
+                                    op: LT
+                                    value: "phone"
+                                  }
+                                   sub_filters {
+                                    name: "gender"
+                                    op: EQUAL
+                                    value: "2"
+                                  }
+                                }
+                                node {
+                                  name: "cookied_pool-GB-phone-Demo Bucket - F18-24"
+                                  index: 0
+                                  branch_node {
+                                    branches {
+                                      chance: 0.1930248
+                                      node{
+                                        population_node {
```

Comment:
population_node need a random_seed. Same below.

Reply:
What should the random_seed be named?